ORStools/proc/isochrones_layer_proc.py

```diff
@@ -210,34 +121,34 @@ def processAlgorithm(self, parameters, context, feedback):
                 "TypeError: Multipoint Layers are not accepted. Please convert to single geometry layer.")
 
         # Get ID field properties
-        id_field_name = self.parameterAsString(parameters, self.IN_FIELD, context)
-        id_field_id = source.fields().lookupField(id_field_name)
+        id_field_name = parameters[self.IN_FIELD]
+        id_field_id = source.fields().indexOf(id_field_name)
```

Comment:
as above, this slightly changes the logic

Reply:
see above :)